caleb-lab/lib/index.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
As far as I can tell, you're not really using this module, so don't include code you're not actually utilizing.